R/sidebar.R

```diff
@@ -122,8 +135,12 @@ sidebar <- function(
       class = c("sidebar", class),
       hidden = if (open == "closed") NA,
       tags$div(
-        class = "sidebar-content",
+        class = "sidebar-content bslib-gap-spacing",
         title,
+        style = css(
+          "--bslib-mb-spacing" = validateCssUnit(gap),
+          "--bslib-sidebar-content-padding" = validateCssPadding(padding)
```

Comment:
I might be missing something, but I think we want to avoid CSS variables here since we don't want a parent `layout_sidebar()` to influence the defaults for a child.

```suggestion
          gap = validateCssUnit(gap),
          padding = validateCssPadding(padding)
```

Reply:
That makes sense, I'll update this.